src/backend/nodes/cypher_readfuncs.c

```diff
@@ -236,7 +236,7 @@ void read_cypher_update_information(struct ExtensibleNode *node)
     READ_LOCALS(cypher_update_information);
 
     READ_NODE_FIELD(set_items);
-    READ_UINT_FIELD(flags);
+    READ_INT_FIELD(flags);
```

Comment:
Why were these changed back to INT from UINT. They were purposely changed to UINT in the master.